bundles/io.openliberty.tools.eclipse.ui/META-INF/MANIFEST.MF

```diff
@@ -7,12 +7,16 @@ Bundle-SymbolicName: io.openliberty.tools.eclipse.ui;singleton:=true
 Bundle-Version: 23.0.9.qualifier
 Bundle-Activator: io.openliberty.tools.eclipse.LibertyDevPlugin
 Export-Package: io.openliberty.tools.eclipse;x-friends:="io.openliberty.tools.eclipse.tests",
+ io.openliberty.tools.eclipse.debug;x-friends:="io.openliberty.tools.eclipse.tests",
  io.openliberty.tools.eclipse.ui.dashboard;x-friends:="io.openliberty.tools.eclipse.tests",
  io.openliberty.tools.eclipse.ui.launch;x-friends:="io.openliberty.tools.eclipse.tests",
  io.openliberty.tools.eclipse.ui.launch.shortcuts;x-friends:="io.openliberty.tools.eclipse.tests",
  io.openliberty.tools.eclipse.ui.preferences;x-friends:="io.openliberty.tools.eclipse.tests"
 Require-Bundle: org.eclipse.ui,
- org.eclipse.equinox.preferences
+ org.eclipse.equinox.preferences,
+ org.eclipse.swt,
+ org.eclipse.m2e.maven.runtime,
+ org.eclipse.m2e.core
```

Comment:
Do you still need the RB here?  Or could these now be removed with the Import-Pkg?

Reply:
OK.. it's OK to use the Require-Bundle for the two m2e bundles.

As far as the other two...it sounds like you're saying that you can't easily see at compile time what classes/pkgs are needed to be imported?   That in and of itself is arguably not a reason to use RB vs ImportPkg.   Is it just one or two packages ?    I could see trying once, twice, and then if there's a third just saying let's use RB?    

However we do it can we avoid dups across RB and IP?   Eg. the IP for m2e should be removed and maybe the swt custom one?